devimint/src/main.rs

```diff
@@ -1870,7 +1895,6 @@ async fn handle_command() -> Result<()> {
                 async move {
                     let dev_fed = dev_fed(&process_mgr).await?;
                     tokio::try_join!(
-                        dev_fed.fed.pegin(10_000),
```

Comment:
Why this?
How an user of `just mprocs` will get ecash?

Reply:
Good catch. I was looking to completely remove dev fed internal_client but after this PR, we still need it for just mprocs and load tests